package.json

```diff
@@ -28,13 +35,71 @@
   "dependencies": {
     "@zeit/next-css": "^1.0.1",
     "express": "^4.16.4",
-    "lru-cache": "^4.1.3",
     "next": "^7.0.2",
     "react": "^16.5.2",
     "react-dom": "^16.5.2"
   },
   "devDependencies": {
+    "babel-eslint": "^10.0.1",
+    "eslint": "^5.7.0",
+    "eslint-config-prettier": "^3.1.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-node": "^7.0.1",
+    "eslint-plugin-prettier": "^3.0.0",
+    "eslint-plugin-promise": "^4.0.1",
+    "eslint-plugin-react": "^7.11.1",
+    "eslint-plugin-standard": "^4.0.0",
+    "jest": "^23.6.0",
     "next-workbox-webpack-plugin": "^1.1.0",
+    "npm-run-all": "^4.1.3",
+    "prettier": "1.14.3",
+    "prettier-eslint": "^8.8.2",
     "webpack-pwa-manifest": "^3.7.1"
+  },
+  "eslintConfig": {
+    "env": {
+      "browser": true,
+      "node": true,
+      "es6": true
+    },
+    "parser": "babel-eslint",
+    "settings": {
+      "react": {
+        "createClass": "createReactClass",
+        "pragma": "React",
+        "version": "16.0"
+      },
+      "propWrapperFunctions": [
+        "forbidExtraProps"
+      ]
+    },
+    "extends": [
+      "eslint:recommended",
+      "plugin:prettier/recommended",
+      "plugin:react/recommended"
+    ],
+    "plugins": [
+      "prettier",
+      "react"
```

Comment:
I don't think you need the react plugin here since you're using the recommended extends.plugin.

Reply:
I removed it and it worked! :)